src/threading.ts

```diff
@@ -117,12 +118,33 @@ class Thread {
     public writeTraceback(): void {
         this.traceback = "Traceback (most recent call last):\n" + this.traceback;
         process.stdout.write(this.traceback);
+        this.exit();
     }
 
-    // Output the traceback when thread cannot handle an exception, and exit the thread
-    public throwException(): void {
-        this.writeTraceback();
-        this.exit();
+    public throwException(f: Py_FrameObject): void {
```

Comment:
Ohh I see. When we standardize the interface for frames across languages, it's going to take an exception object. But I'll also have to figure out how to reconcile the fact that JVM exception handling must be asynchronous, since type checking may trigger loading in a class....

So, for now, you can implement `tryCatchException` without an exception argument.